SwiftCompilerSources/Sources/SIL/Test.swift

```diff
@@ -0,0 +1,136 @@
+//===----------- Test.swift - In-IR tests from Swift source ---------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2022 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import Basic
+import SILBridging
+
+@_semantics("arc.immortal")
+public class FunctionTest {
```

Comment:
Does this really need to be a class? And does it need to bridge to C++ (via the swift object header)?
On the C++ side, a new FunctionTest is created anyway.
I think it can be a struct, similar to `FunctionPass`

Reply:
Thanks!  Switched over to that.